mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -32,6 +32,8 @@ class TunableDict(TypedDict, total=False):
     default: TunableValue
     values: Optional[List[Optional[str]]]
     range: Optional[Union[Sequence[int], Sequence[float]]]
+    quantization: Optional[int]
```

Comment:
Per discussion above:
- floats accept quantization as a float > 0, but
- ints accept quantization as an int

Reply:
mm, I cannot see a `float` anywhere. Parameter `q` is an integer number of the intervals to split the hyperparameter's range into. `ConfigSpace` API requires it to be either `int` or `None`
Here's a factory method I invoke for float hyperparameters: https://automl.github.io/ConfigSpace/main/_modules/ConfigSpace/api/types/float.html

Reply:
**UPDATE:** I've found where `q` can be `float` - it's only when we pass it directly to the constructor of a specific distribution-specific float hyperparameter. I never do that. I use the `ConfigSpace.Float()` factory method instead, and it only takes `q` as `int` - even though all three classes that it can build can take a `float`. I would suggest to keep it as `int` for simplicity - otherwise our code will get bloated by duplicating the `ConfigSpace.Float()` functionality

Reply:
Added a PR I think addresses this here:
https://github.com/motus/MLOS/pull/10

Reply:
merged, thank you!